docs/studios/unity/tutorials/github_actions.md

```diff
@@ -0,0 +1,309 @@
+---
+sidebar_label: 'Testing with GitHub Actions'
+---
+
+# Setting up QA Bots with GitHub Actions
+
+This tutorial will walk you through setting up a QA Bot for your Unity project using GitHub Actions. This tutorial assumes you have
+some familiarity with [GitHub Actions](https://docs.github.com/en/actions), and that you have a working bot within your
+game (see more on this within the [Creating your first bot guide](/studios/unity/unity-sdk/creating-bots/configuration)). 
+You can find a working example in [this repository](google.com) for reference.
+
+:::caution
+
+This guide and reference is subject to change, as the product is in an **early preview**. We
+are making changes based on your recommendations and feedback, so let us know what you think at
+info@regression.gg, or join our [Discord](https://discord.com/invite/925SYVse2H).
+
+:::
+
+:::info
+
+This features requires a license for Unity, since CI/CD builds for Unity are only available if you have
+a license for Unity. Additionally, if you want to use your existing CI/CD pipelines, the best reference
+here is the environment variable and test function sections below.
+
+:::
+
+## Overview
+
+In this guide, we will use GitHub Actions, GameCI, and Regression Games to set up a QA Bot for your Unity project. Whenever a new
+push is made within your project, the Unity project will be built and a bot will be started. The steps to this
+guide are as follows:
+
+1. Add the GitHub Actions workflow files to your project needed to get Unity builds to work.
+2. Configuring the GitHub Actions workflow files to work with GameCI and Regression Games.
+3. Add a test to your Unity project that will start and run your bot.
+4. Learn more about how to view test results.
+
+## Add GitHub Action Workflow Files
+
+We use the [GameCI](https://game.ci/) project as a foundation for running GitHub CI/CD builds for Unity. The GameCI project provides a collection of GitHub Actions that are used to build Unity projects.
+
+#### 1. Create a new branch in your project called `qa-tests`.
```

Comment:
is this branch-name required ?  What is the significance of this specific name.  We should explain that to them and/or let them know that they can use their own branch name and what other settings need to adjust to accommodate it.